spec/lib/catalog/rbac/access_spec.rb

```diff
@@ -59,32 +62,12 @@
       let(:rbac_enabled) { true }
 
       before do
-        allow(catalog_access).to receive(:scopes).with("portfolio_items", verb).and_return(scopes)
+        allow(catalog_access).to receive(:scopes).with(aceable_type.table_name, verb).and_return(scopes)
```

Comment:
I see this change clarifies why we needed the PR fix, but only because of the PR description. But It would be great if it is possible to spec out the converse that no longer happens because of this PR.